lib/beaker/options/parser.rb

```diff
@@ -228,10 +228,22 @@ def parse_hosts_options
         hosts_options = Beaker::Options::HostsFileParser.parse_hosts_file(@options[:hosts_file])
         return hosts_options
       rescue Errno::ENOENT
-        require 'tempfile'
+        error_message = "\nHosts file '#{@options[:hosts_file]}' does not exist."
+        error_message << "\nTrying as beaker-hostgenerator input.\n\n"
+        $stderr.puts error_message
```

Comment:
This isn't an error that it's not a hosts file, so printing to stderr seems wrong.  This is a supported feature, and normal behavior, so stdout seems appropriate.